### ISSUE TITLE
MD026 - trailing punctuation in header - allow

### ISSUE THREAD

**Issue by steven7mwesigwa** [DavidAnson/vscode-markdownlint#54]
i would like to allow use of ? in header

In my User settings i set it to:

```
"markdownlint.config": {
	"MD013": false,
	"MD026":{".,;:!"}
}
```
Though it is warning me that it expects a `boolean` or `object` for "MD026".
I know am missing something, but  i don't know what.

**Comment by DavidAnson** [comment]
VS Code will help you get the schema right by prompting you with valid values. (You can force it with Control-Space.) In this case, you need to use the `punctuation` parameter, so it should look something like the following:

    “MD026”: { “punctuation”: “.,;:!”}

Example: https://github.com/DavidAnson/markdownlint/blob/2710c375b3c9b9b6531d546b667f50f7d0c956f2/test/heading_trailing_punctuation_customized.json

Documentation: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md026

**Comment by DavidAnson** [comment]
Cheers!
